section 4_Flask_Restfull/app.py

```diff
@@ -0,0 +1,105 @@
+'''This is the main file which contains all the end points'''
+
+
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+from flask_jwt import JWT, jwt_required
+from security import authenticate, identity
+import enum
+app = Flask(__name__)
+app.secret_key = 'sdfdslfls@#1223$lskd'
+api = Api(app)
+
+jwt = JWT(app,authenticate, identity) # gives /auth end point
+
+items: list = []
+
+
+
+
+
+
+
+# Using enum class create enumerations
+class http_status(enum.Enum):
+   Ok = 200
+   Created = 201
+   Accepted = 202
+   BadRequest = 400
+   Unauthorized = 401
+   NotFound = 404
```

Comment:
No need to create this, you can use this instead.

`from http import HTTPStatus`